www-apps/gitea/gitea-1.7.1.ebuild

```diff
@@ -31,21 +31,31 @@ pkg_setup() {
 src_prepare() {
 	default
 	sed -i -e "s/\"main.Version.*$/\"main.Version=${PV}\"/"\
-		-e "s/-ldflags '-s/-ldflags '/" src/${EGO_PN}/Makefile || die
+		-e "s/-ldflags '-s/-ldflags '/" \
+		-e "s/GOFLAGS := -i -v/GOFLAGS := -v/" \
```

Comment:
Why removing -i from GOFLAGS?

Reply:
The `-i` flag seems to be responsible for some cases of GO stdlib recompilation, causing permissions errors, I encountered. Described in this issue: https://github.com/golang/go/issues/24674#issuecomment-379085520

By the way, `-i` is convenient when developing, as it speed up futhers recompilations, there is no gain in our case as we only compile one time.